src/document_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'telegramAPI'
+
+class DocumentHandler
+  attr_reader :document, :api, :url
+
+  def initialize(request_handler)
+    @token = ENV['TOKEN']
+    @document = request_handler.message.document
+
+    @api = TelegramAPI.new(@token)
+    @url = "https://api.telegram.org/file/bot#{@token}/#{api.getFile(document.file_id)['file_path']}"
+  end
+
+  def translate_document
+    File.new('translated.txt', 'w').puts translate_content
+  end
+
+  private
+
+  def get_file_content
+    puts api.getFile(document.file_id).fetch('file_path')
+    open(url) do |f|
```

Comment:
Add `require 'open-uri'` at the top of the file.
Please, verify your code — I don't want to debug it :)